README.md

```diff
@@ -40,6 +40,8 @@ To make a contribution follow the steps laid out in the [beginner's guide on how
 https://cosima-recipes.readthedocs.io/en/latest/contributing.html). If they sound intimidating then don't worry!
 Just raise [an issue](https://github.com/COSIMA/cosima-recipes/issues) explaining briefly what the contribution you want to make is and we'll help out with the process!
 
+To acknowledge contributions, contributors to the COSIMA Cookbook are added to the **citable DOI** entry associated with the repository. Hence, users who put together a pull request for a new contribution (starting with an issue -- as above), should ensure that the pull request also modifies the .zenodo.json file to include their affiliation details.
```

Comment:
I think it's too wordy. How about chopping the first clause? So it becomes:

> Contributors to the COSIMA Cookbook are added to the **citable DOI** entry

Reply:
Yes, agreed!